syncopy/shared/computational_routine.py

```diff
@@ -596,18 +597,29 @@ class method (starting with the word `"compute_"`).
 
         # Do not spill trials on disk if they're supposed to be removed anyway
         if parallel_store and not self.keeptrials:
-            msg = "trial-averaging only supports sequential writing!"
+            msg = "trial-averaging only supports sequential writing, disabling parallel mode!"
             SPYWarning(msg)
             parallel_store = False
 
         # Create HDF5 dataset of appropriate dimension
         self.preallocate_output(out, parallel_store=parallel_store)
 
-        # Concurrent processing requires some additional prep-work...
         if parallel:
 
             # Construct list of dicts that will be passed on to workers: in the
             # parallel case, `trl_dat` is a dictionary!
+
+            # Compute trial_ids and chunk_ids to turn into a unique index.
+            # Keep in mind that these may not be absolute due to selections.
```

Comment:
oh, I was not aware of that

Reply:
Ya me neither when we discussed it, but now when browsing through the code I realized that the CR of course has access to it.. Not sure, we are in no rush, so if you want to make use of this, let me know.. otherwise it's also fine for me to merge anyways!